### PR TITLE
Dynamic bubbles

### PR DESCRIPTION
**Pull request by chenikabukes**
Created GetWindowSize.js to get window size of user's browser.
Set size of bubbles to ensure no overflow out of SVG object.
Make force in ForceGraph.js dynamic according to window size attributes.

### CODE REVIEW

File: client/src/pages/Dashboard/components/D3BubbleChart.js [hunk 1]
```diff
@@ -1,43 +1,56 @@
...
+
     if (size === 0) {
         return (<p></p>)
     }
     else {
         return (
-            <div>
-                <svg width="1600" height="600">
-                    <ForceGraph nodes={data.datasets} maxRadius={/*maxSoFar*/ 60} />
+            <div width="1500" height="1000">
```
Comment (Dhairya-Khara) [inline client/src/pages/Dashboard/components/D3BubbleChart.js:51]: These numbers shouldnt be hard coded right?

**Review by Dhairya-Khara — CHANGES_REQUESTED** [review]
image

The bubbles are very small, lets make them a bit bigger.

image

The bubbles are also way to the right. Let's center them for now.

File: client/src/pages/Dashboard/components/D3BubbleChart.js (hunk 1 above)
Comment (chenikabukes) [reply client/src/pages/Dashboard/components/D3BubbleChart.js:51]: That's the old code, look below in the changed file for the updated code.

**Comment by BerkeAltiparmak** [comment]
Everything looks great. It seems many issues are fixed, and there are more things to do (such as ensuring the bubbles fit in the area given to them, because it still seems to be an issue), but those are not quick changes and can be implemented in the future PRs. I'm merging this, it seems to achieve everything it promises to achieve in a clean way.

**Pull request merged**